src/c-writer.cc

```diff
@@ -3346,8 +3421,13 @@ void CWriter::Write(const StoreExpr& expr) {
 
   Memory* memory = module_->memories[module_->GetMemoryIndex(expr.memidx)];
 
-  Write(func, "(", ExternalInstancePtr(ModuleFieldType::Memory, memory->name),
-        ", (u64)(", StackVar(1), ")");
+  if (options_.no_sandbox) {
```

Comment:
Same comment here (could this be kept the same, given the changes to wasm2c.declarations.nosandbox.c)?